src/QMCWaveFunctions/BsplineFactory/createComplexDouble.cpp

```diff
@@ -37,44 +37,62 @@ BsplineReaderBase* createBsplineComplexDouble(EinsplineSetBuilder* e, bool hybri
   BsplineReaderBase* aReader = nullptr;
 
 #if defined(QMC_COMPLEX)
+  app_summary() << "    Using complex valued spline SPOs with complex double precision storage (C2C)." << std::endl;
 #if defined(ENABLE_OFFLOAD)
   if (useGPU == "yes")
   {
     if (hybrid_rep)
     {
-      app_log() << "OpenMP offload has not been enabled with hybrid orbital representation!"
-                << " Running on the host." << std::endl;
+      app_summary() << "OpenMP offload has not been implemented to support hybrid orbital representation!"
+                    << " Running on CPU." << std::endl;
+      app_summary() << "    Using hybrid orbital representation." << std::endl;
       aReader = new HybridRepSetReader<HybridRepCplx<SplineC2C<double>>>(e);
     }
     else
+    {
+      app_summary() << "    Running on an accelerator via OpenMP offload." << std::endl;
```

Comment:
For spline SPO, print "Running on an accelerator via OpenMP offload." here and "Running on CPU" below